tests/cupy_tests/sparse_tests/test_linalg.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+
+import numpy
+import scipy.sparse.linalg  # NOQA
```

Comment:
Check if `scipy` is available. I mean use `try-catch` and if scipy is not available, skip test.